accounts/usbwallet/hub.go

```diff
@@ -156,11 +162,13 @@ func (hub *Hub) refreshWallets() {
 	}
 	infos, err := usb.Enumerate(hub.vendorID, 0)
 	if err != nil {
+		failcount := atomic.AddUint32(&(hub.enumFails), 1)
 		if runtime.GOOS == "linux" {
 			// See rationale before the enumeration why this is needed and only on Linux.
 			hub.commsLock.Unlock()
 		}
-		log.Error("error enumerating USB enumeration: ", "code", err)
+		log.Error("Failed to enumerate USB devices", "hub", hub.scheme,
+			"vendor", hub.vendorID, "failcount", failcount, "err", err)
 		return
 	}
```

Comment:
Question: do we want to reset the counter if enumeration succeeds? That way occasional flukes don't disable the wallets.

Reply:
Btw, you don't need paranthesis around `&(hub.enumFails)`

Reply:
No. I'll add that